drivers/wifi-client/driver.js

```diff
@@ -84,6 +86,56 @@ class UnifiDriver extends Homey.Driver {
         }, (12 * 3600 * 1000)))
     }
 
+    getUsageData(){
+        this._debug('Get usage data for devices');
+
+        this.unifi.get("stat/alluser").then((rawdata)=>{
+            this._debug("Retrieved statistics for connected devices");
+            let devices = rawdata.data;
+
+            let mappedData = new Map();
+
+            devices.forEach(entry =>{
+               mappedData.set(entry.mac, entry);
+            });
+
+            this.getDevices().forEach(device =>{
+                let deviceData = device.getData();
+                let tx = 0;
+                let rx = 0;
+
+
+                if(mappedData.has(deviceData.id)){
+                    let unifyData = mappedData.get(deviceData.id);
+                    if(unifyData.hasOwnProperty('tx_bytes')){
+                        tx = unifyData.tx_bytes / GB_DIVISOR;
+                    }
+                    if(unifyData.hasOwnProperty('rx_bytes')){
+                        rx = unifyData.rx_bytes / GB_DIVISOR;
+                    }
+                }
+
+                device.setCapabilityValue("measure_bytes_received",tx,(err,result)=>{
```

Comment:
Updating values should be done through the device driver in the unifi app. Preferably by either creating a new method in the device driver, or by incorporating already existing methods (and at least by by using private _updateProperty method, which is updating only if data has changed.)